Common/Interfaces/INewsPublisher.cs

```diff
@@ -9,6 +8,6 @@ namespace Interfaces
     /// </summary>
     public interface INewsPublisher
     {
-        List<ExchangeRate> GetCurrencies();
+        IEnumerable<NewsItem> GetNews(string feedUrl);
     }
 }
```

Comment:
Why you use IEnumerable<T> and not specific type?

Reply:
> Why you use IEnumerable and not specific type?

Hm, I made it IEnumerable because I did not see any reason to return more complicated class such as List here, but I might be wrong =)